modules/dev-tools/docs/cli/ocular-publish.md

```diff
@@ -18,6 +18,9 @@ npx ocular-publish [mode] [npm-tag]
 
 - `beta` - bump pre-release version and publish with beta flag.
 - `prod` - bump patch version and publish.
+- `version-beta` - bump pre-release version only.
```

Comment:
These docs looks quite terse, perhaps we can be a bit more verbose. Without reading the source I can't quite tell what these do.

Nit: The docs make it look like these two modules don't actually publish? If that is true, should we call them ocular-bump instead of ocular-publish then? or maybe `ocular-publish bump-beta` or `ocular-publish bump beta` `ocular-publish bump-only beta`

Reply:
In npm, yarn, and lerna, the "version" verb means incrementing the release number, and the "publish" verb means pushing that change to a registry. A postversion script might also run publish if projects prefer to do it all in one step, which seems common. 

I'd prefer if ocular could reuse these terms as much as possible — "bump" has no precise meaning to me. 

I do see your point that `ocular-publish version-beta` feels unclear, though...

Reply:
Can't use two words as the second argument is currently used as a custom NPM dist tag. This can of course change because it's not actively used by any repo and it seems like we are going with `publishConfig.tag` instead (I think that's a better approach).

Otherwise, does `version-only-beta` sound better?